.github/workflows/deployDemo.yml

```diff
@@ -50,11 +50,15 @@ jobs:
       - name: Install dependencies
         run: yarn install
       - name: Build deploy artifact
+        env:
+          REACT_APP_BASE_URL: https://demo.simplereport.gov
+          REACT_APP_BACKEND_URL: https://demo.simplereport.gov/api
+          PUBLIC_URL: /app/
```

Comment:
Seems like this could live in `.env.production`? In Spring we could probably avoid the repetition of the base and backend URLs, but I doubt it would work here.

Tangential thought: we are using path-based dispatch because we've always used it, but we don't _have_ to have `/api` exist on the same domain any more, do we? That was just because we didn't have the ability to have a wildcard cert or a three-dot host name before we had our own domain.

Reply:
Yes, I really want to move us away from the path based dispatch